can/io/mf4.py

```diff
@@ -0,0 +1,160 @@
+# coding: utf-8
+
+"""
+Contains handling of MF4 logging files.
+
+"""
+
+from __future__ import absolute_import
+
+from datetime import datetime
+from pathlib import Path
+import logging
+
+from ..listener import Listener
+from ..util import channel2int
+from .generic import BaseIOHandler
+
+from asammdf import MDF, Signal
```

Comment:
We could surround this with a try/except block that sets `ASAMMDF_AVAILABLE = True` and wrap the `class MF4Writer ...` in an `if ASAMMDF_AVAILABLE:`-block. This unties all of this from version-checking and uses feature-checking instead.